src/common/viz/PlotlyDescExtractor.js

```diff
@@ -48,11 +48,14 @@ define([], function () {
             ylabel: node.getAttribute('ylabel'),
         };
 
-        const lineIds = node.getChildrenIds();
-        desc.lines = lineIds.map((lineId) => {
-            let lineNode = this._client.getNode(lineId);
-            return this.getLineDesc(lineNode);
-        });
+        const childIds = node.getChildrenIds();
+        desc.lines = childIds
+            .map(childId => this._client.getNode(childId))
+            .filter((node) => node.getValidAttributeNames().includes('points'))
+            .map(lineNode => this.getLineDesc(lineNode));
+        desc.images = childIds.map(childId => this._client.getNode(childId))
```

Comment:
These duplicate `map` calls could be removed by calling them on `node.getChildrenIds()` rather than for both the lines and images:
```javascript
const children = node.getChildrenIds().map(id => this._client.getNode(id));
desc.lines = children
    .filter(node => ....)
```